pennylane/transforms/mitigate.py

```diff
@@ -318,6 +324,46 @@ def richardson_extrapolate(x, y):
     return poly_extrapolate(x, y, len(x) - 1)
 
 
+def exponential_extrapolate(x, y, asymptote=None, eps=1.0e-6):
+    r"""Exponential fitting
```

Comment:
Could you add some more docs about exponential extrapolate, your PR description would be useful here.

Reply:
Yup, meant to do that! Thanks: https://github.com/PennyLaneAI/pennylane/pull/5972/commits/113aaa881e39854c3032dceefcb8588c5927d3b3